collections/trad-japanese-blue/inscriptions.json

```diff
@@ -0,0 +1,178 @@
+[
+  {
+    "id": "cd4fd42952aa0f7ae5ef6697d13081637b8e84b76deb69d0dc2778cc269b5de0i0",
+    "meta": {
+      "id": 1,
+      "creator": "0xKyosuke",
+      "name": "み空色 -Misorairo-",
```

Comment:
hey i didnt notice this on the red collection but can you follow the format of metadata in the readme